src/main/java/com/robosh/ejournal/data/dto/teacher/TeacherDto.java

```diff
@@ -0,0 +1,21 @@
+package com.robosh.ejournal.data.dto.teacher;
+
+import com.robosh.ejournal.data.dto.UserDto;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class TeacherDto extends UserDto {
+
+    protected String description;
+
```

Comment:
protected?

Reply:
i think lately we will extend this by class like teacher with group dto, so that i made it protected

Reply:
Ok